examples/interop/README.md

```diff
@@ -137,6 +137,13 @@ Run below command inside each application folder.
 
```

Comment:
[This step](https://github.com/LiskHQ/lisk-sdk/blob/6f54c99f5d81007a31cfa4f73d671166acb888b4/examples/interop/README.md#transfer-sidechain-two-to-sidechain-one) doesn't work as there is no such file in the mentioned folder. Please update either the readme or add the missing code file.